internal/service/streamprocessor/data_source_test.go

```diff
@@ -9,23 +9,33 @@ import (
 	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
 )
 
-var dataSourceName = "data.mongodbatlas_streamprocessor.test"
+var dataSourceName = "data.mongodbatlas_stream_processor.test"
+var pluralDataSourceName = "data.mongodbatlas_stream_processors.test"
 
 func TestAccStreamProcessorDS_readManual(t *testing.T) {
```

Comment:
Unrelated to this PR, but I believe one of @lantoli's initiative to improve our testing infra was to "merge" data source testing with the resource itself to avoid creating ad-hoc resources only for the DS. Has that been applied here too? Could it be applied?

Reply:
Yes, this is what the comment on line 16 is about. Create [issue here](https://jira.mongodb.org/browse/CLOUDP-268552)